extension/content.js

```diff
@@ -51,6 +52,24 @@ function addReleasesTab() {
 	}
 }
 
+function addBlameParentLinks() {
+	let pathname = location.pathname;
+
+	$('.blame-sha').each(function(index, commitLink) {
+		let $commitLink = $(commitLink)
+		let $blameParentLink = $commitLink.clone();
+		let commitSha = commitLink.href.match(/(?<=commit\/)\w+/)[0];
```

Comment:
You're cloning the link in the DOM, which is the full sha hash and nothing after it. At least that's what I found. If you were taking it from the `location.pathname` then you're probably right about other user stuff.